### DISCUSSION TITLE
Fragmented E01

### DISCUSSION THREAD

**Discussion by thiagocaparelli** [sepinf-inc/IPED#1683] [Q&A] [answered]
Hello, I've a fragmented 10TB image (size 200GB).
I've tried to process the forensic image like this:

iped.exe -d j:Disco001.E01 -o D:\Caparelli\SP198-2023\Dados-Processados-IPED-Disco001 --portable -profile forensic

Occour that the IPED ignored the others files and processed just the first one (001).
The files doesn't have sequential timestamp maybe is this.
What can I do?

![GIT](https://github.com/sepinf-inc/IPED/assets/55517932/647cdcbe-9633-443c-a89d-dc72a18c27b6)

**Comment by lfcnassif** [comment]
The normal behavior is processing all of them, but just the first fragment is shown into the case. If the file system was decoded correctly, you can be confident the other segments were processed fine. You can also check the partitions size in the Evidence tree tab.

**Comment by gisms** [comment, marked as answer]
considering the doubt arising from the timestamp of the parts, you can check the integrity of the image usign ftk imager

**Reply by thiagocaparelli** [reply]
In deed. For some reason when I copied the data to storage NAS it corrupted the image. Thank you so much.
